### ISSUE TITLE
PeFT Training is throwing the following warning

### ISSUE THREAD

**Issue by shivajid** [google/tunix#136]
## Expected Behavior
No warnng
## Actual Behavior

WARNING:absl:Metric learning_rate not found for mode train. Not logging metric.

## Steps to Reproduce the Problem

1. Run the qlora_demo.ipynb example
1.
1.

## Specifications

- Version:
- Platform: TPU v6e

**Comment by tianshub** [comment]
Thanks for reporting, this will be fixed.
